ogre2/include/ignition/rendering/ogre2/Ogre2RenderEngine.hh

```diff
@@ -175,9 +175,13 @@ namespace ignition
       /// \return a list of FSAA levels
       public: std::vector<unsigned int> FSAALevels() const;
 
+      /// \brief Deprecated. Use SphericalClipMinDistance instead
+      public: Ogre2IgnHlmsSphericalClipMinDistance IGN_DEPRECATED(7) &
```

Comment:
can you add an entry to the [Migration.md](https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering6/Migration.md) guide?

Reply:
I'll add this in a separate PR

Reply:
https://github.com/ignitionrobotics/ign-rendering/pull/538